packages/ui/src/components/logo/logo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import { LogoNameMap } from './logo-name-map'
+
+interface LogoProps {
+  name: keyof typeof LogoNameMap
+  size?: number
+  original?: boolean // Uses brand color when true
+}
+
+const Logo: React.FC<LogoProps> = ({ name, size = 24, original = false }) => {
+  const icon = LogoNameMap[name]
```

Comment:
What happens if `name` isn't in the map? Might be a good idea to check for that and render a blank SVG or return a fragment?

Reply:
Done, returning a `<></>` if name is not in map.